include/deal.II/particles/data_out.h

```diff
@@ -0,0 +1,100 @@
+// ---------------------------------------------------------------------
+//
+// Copyright (C) 2017 - 2019 by the deal.II authors
+//
+// This file is part of the deal.II library.
+//
+// The deal.II library is free software; you can use it, redistribute
+// it, and/or modify it under the terms of the GNU Lesser General
+// Public License as published by the Free Software Foundation; either
+// version 2.1 of the License, or (at your option) any later version.
+// The full text of the license can be found in the file LICENSE.md at
+// the top level directory of deal.II.
+//
+// ---------------------------------------------------------------------
+#ifndef dealii_particles_data_out_h
+#define dealii_particles_data_out_h
+
+#include <deal.II/base/config.h>
+
+#include <deal.II/base/data_out_base.h>
+#include <deal.II/base/mpi.h>
+
+#include <deal.II/particles/particle_handler.h>
+
+#include <string>
+#include <vector>
+
+
+DEAL_II_NAMESPACE_OPEN
+
+namespace Particles
+{
+  /**
+   * This class manages the DataOut of a Particle Handler
+   * From a particle handler, it generates patches which can then be used to
+   * write traditional output files. This class currently only supports witing
+   * the particle position and their ID and does not allow to write the
+   * properties attached to the particles
+   *
+   * @ingroup Particle
+   *
+   * @author Bruno Blais, Luca Heltai 2019
+   */
+  template <int dim, int spacedim>
+  class DataOut : public dealii::DataOutInterface<0, spacedim>
+  {
+  public:
+    /**
+     *Default constructor for the Particles::DataOut class.
+     */
+    DataOut() = default;
+
+    /**
+     *Default destructor for the Particles::DataOut class.
```

Comment:
There can only be one :-)
```suggestion
     * Destructor for the Particles::DataOut class.
```